circuits/cpp/src/aztec3/circuits/kernel/private/native_private_kernel_circuit_ordering.test.cpp

```diff
@@ -31,6 +31,52 @@ class native_private_kernel_ordering_tests : public ::testing::Test {
     static void SetUpTestSuite() { barretenberg::srs::init_crs_factory("../barretenberg/cpp/srs_db/ignition"); }
 };
 
+// TODO(1998): testing cbind calls private_kernel__sim_ordering, private_kernel__sim_init, private_kernel__sim_inner
+//       in their respective test suites once msgpack capabilities allow it. One current limitation is due to
+//       the lack of support to deserialize std::variant in particular CircuitResult type.
+//       See https://github.com/AztecProtocol/aztec-packages/issues/1998
+/**
+ * @brief Test cbind
+ */
+// TEST_F(native_private_kernel_ordering_tests, cbind_private_kernel__sim_ordering)
```

Comment:
Similar to above, better to disable a test than comment it out.

Reply:
Yep, but it does not disable compilation right? This is the current issue. Both tests would not compile.